modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/modelEnum.mustache

```diff
@@ -0,0 +1,123 @@
+{{#jackson}}
```

Comment:
when you've time, can you please file another PR to clean up {{#jackson}} ... {{/jackson}} in this file as okhttp-gson library doesn't use the jackson mustache tag?